thewalrus/_permanent.py

```diff
@@ -15,29 +15,24 @@
 Permanent Python interface
 """
 import numpy as np
-
+from numba import jit
 from ._hafnian import hafnian_repeated
-from .libwalrus import perm_complex, perm_real, perm_BBFG_real, perm_BBFG_complex
 
+# pylint: disable = C0103, R0914
 
-def perm(A, quad=True, fsum=False, method="bbfg"):
-    """Returns the permanent of a matrix using the `method` formula
 
+def perm(A, method="bbfg"):
+    """Returns the permanent of a matrix using the `method` formula
     For more direct control, you may wish to call :func:`perm_real`
     or :func:`perm_complex` directly.
-
     Args:
         A (array): a square array.
```

Comment:
is array allowed to be complex?

Reply:
Yes! We tried with complex arrays and it worked

Reply:
@josh146 : do you mean that we should make explicit that it can be float or complex?

Reply:
yep! just something like 
```suggestion
        A (array[float or complex]): a square array.
```